api/docgen/cmd/docgen.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+
+	"github.com/celestiaorg/celestia-node/api/docgen"
+	"github.com/celestiaorg/celestia-node/nodebuilder"
+)
+
+func main() {
+	// 1. Parse the module names from the command line arguments
+	var moduleNames = os.Args[1:]
+
+	// 2. Open the respective nodebuilder/X/service.go files for AST parsing
```

Comment:
```suggestion
	// 2. Open the respective nodebuilder/X/module.go files for AST parsing
```

Reply:
incorrect actually

Reply:
and this changes in the service renaming PR